--- Web/View/Static/Welcome.hs
+++ Web/View/Static/Welcome.hs
@@ -15,6 +15,8 @@ instance View WelcomeView where
                       It's working!
                   </h2>

+                  <div id="halogen"></div>
+
                   <p style="margin-top: 1rem; font-size: 1.75rem; font-weight: 600; color:hsla(196, 13%, 80%, 1)">
                      Your new application is up and running.
                   </p>
